src/WebJobs.Script/Content/Script/functions.js

```diff
@@ -49,7 +51,15 @@ function createFunction(f) {
         });
         context.log = log;
 
+        if (typeof func === "string") {
```

Comment:
Do we have cases where we're passing in a function here, or is it always a script file path now? If the latter, I'd rename this var to 'scriptFilePath' or something and require/assume it to be a string.